js/interface.js

```diff
@@ -40,7 +40,7 @@ var defaultForm = [
   '</div>',
   '<hr class="hr-normal">',
   '<div class="form-btns clearfix">',
-    '<button type="submit" class="btn btn-primary pull-right">Submit</button>',
+    '<button type="submit" class="btn btn-primary pull-right" data-edit-label="Update">Submit</button>',
```

Comment:
If your form expect to be set in edit mode, then you can optionally provide the label to be displayed when the form is in edit mode.